src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.figure;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculator, Drawable {
```

Comment:
do we need to implement interfaces here?

Reply:
sorry, I forgot to delete them